{"name": "test_get_request_singleusernotfound", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status 404 but got 401\nassert 401 == 404\n +  where 401 = <Response [401]>.status_code", "trace": "self = <test_cases.test_GetRequest.TestGetRequest object at 0x000002161CB916D0>\n\n    def test_get_request_singleusernotfound(self):\n     log = self.get_the_logs()\n     log.info(\"Starting test_get_request_delayedresponse\")\n    \n     # Send GET request\n     response = requests.get(Get_URL3)\n     log.info(f\"GET response: {response}\")\n    \n     # Validate response\n>    assert response.status_code == 404, f\"Expected status 404 but got {response.status_code}\"\nE    AssertionError: Expected status 404 but got 401\nE    assert 401 == 404\nE     +  where 401 = <Response [401]>.status_code\n\ntest_cases\\test_GetRequest.py:95: AssertionError"}, "attachments": [{"name": "log", "source": "26b4398d-c685-4e7b-84e5-48f9863083e8-attachment.txt", "type": "text/plain"}], "start": 1747335020283, "stop": 1747335020683, "uuid": "dc52495e-820d-4ab2-ab00-9f814c69286c", "historyId": "71f5c67e5ce6b42734790dff29c6a116", "testCaseId": "71f5c67e5ce6b42734790dff29c6a116", "fullName": "test_cases.test_GetRequest.TestGetRequest#test_get_request_singleusernotfound", "labels": [{"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_GetRequest"}, {"name": "subSuite", "value": "TestGetRequest"}, {"name": "host", "value": "DESKTOP-FR5AEQ7"}, {"name": "thread", "value": "20176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_GetRequest"}]}